src/FieldCache.php

```diff
@@ -120,7 +120,7 @@ function __filter_graphql_pre_resolve_field(
         $field_name = Utils::sanitize($this->field_name);
         $args_hash = Utils::hash(Utils::stable_string($args));
         $query_hash = Utils::hash($this->query);
-        $user_id = get_current_user_id();
+        $user_id = $this->per_user ? get_current_user_id() : 0;
```

Comment:
Should the fallback be an env var value/configurable and defaulting to 0? Or you think this is OK?

Reply:
Not a bad idea -- I've made it configurable.